Update Panorama.py - URL Filter category updates

epartington opened the pull request:
updated url fitler categories to include missing items financial-services, not-resolved, real-time-detection and ransomware

<!-- REMINDER: THIS IS A PUBLIC REPO DO NOT POST HERE SECRETS/SENSITIVE DATA -->

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [] In Progress
- [X] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.
updated missing url filter categories in the integration (financial-services, not-resolved, real-time-detection and ransomware) 

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [X] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [ ] Documentation

YuvHayun commented:
Hey @epartington, thank you for your contribution!
Please note that you have a failing unit test due to the changes you made.
Also, please update the release note to document the changes you made.
Let me know if you have any questions.

epartington commented:
Updated the version number in the pack file, added a new change file.  the only changes that I made were to the 4 url filter category values that were missing from the integration.. not sure why all the other failed items exist.. those were not touched by my changes

GuyAfik commented on Packs/PAN-OS/ReleaseNotes/1_14_20.md:
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Palo Alto Networks PAN-OS
+- Updated URL Filtering category list to include finanical-services, not-resolved, real-time-detection and ransomeware.
```
```suggestion
- Fixed an issue in the ***pan-os-get-url-category*** command where the *finanical-services*, *real-time-detection* and * *ransomeware* URL categories were not identified.
```

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/demisto/content?pullRequest=21344) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/demisto/content?pullRequest=21344) before we can accept your contribution.<br/>**5** out of **6** committers have signed the CLA.<br/><br/>:white_check_mark: bakatzir<br/>:white_check_mark: merit-maita<br/>:white_check_mark: epartington<br/>:white_check_mark: GuyAfik<br/>:white_check_mark: YuvHayun<br/>:x: aamar88<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/demisto/content?pullRequest=21344) it.</sub>

GuyAfik commented on Packs/PAN-OS/ReleaseNotes/1_14_21.md:
```diff
@@ -0,0 +1,4 @@
+#### Integrations
+##### Palo Alto Networks PAN-OS
+- Fixed an issue in the ***pan-os-get-url-category*** command where the *finanical-services*, *real-time-detection* and * *ransomeware* URL categories were not identified.
```
```suggestion
- Fixed an issue in the ***pan-os-get-url-category*** command where the *finanical-services*, *real-time-detection* and  *ransomeware* URL categories were not identified.
```

GuyAfik replied:
@YuvHayun fixed small typo

YuvHayun commented:
@epartington merging, thx for your contribution!
